regression/contract_to_stub/contract1/stub.c

```diff
@@ -0,0 +1,13 @@
+#include <stub_header.h>
+
+// __stub_stub_impl
+//
+void __stub_stub_impl();
+
+// stub
+// file function.c line 2
+void stub()
+{
+  assert(0 != 0);
```

Comment:
Just `assert(0)`

Reply:
The stub.c files in the regression test directories are generated from the stub making mechanism. I will add them in a local .gitignore.